apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -271,7 +271,7 @@ public LocalContainerEntityManagerFactoryBean entityManagerFactory(DataSource da
   /** @return the {@link Properties} to configure Hibernate and JPA with */
   private Properties jpaProperties() {
     Properties extraProperties = new Properties();
-    extraProperties.put(AvailableSettings.HBM2DDL_AUTO, Action.VALIDATE);
+    extraProperties.put(AvailableSettings.HBM2DDL_AUTO, Action.NONE);
```

Comment:
Won't having this same change give you a merge conflict? Or is Git / GitHub smart enough to avoid that?

Reply:
I believe GItHub will be smart enough after we rebase.